CesiumGS/cesium
# Update npm packages

# Description

`npm outdated` ->
```
Package           Current  Wanted  Latest  Location                       Depended by
jsdoc              3.6.11  3.6.11   4.0.4  node_modules/jsdoc             cesium
lerc                2.0.0   2.0.0   4.0.4  node_modules/lerc              engine@npm:@cesium/engine@12.0.1
markdownlint-cli   0.42.0  0.42.0  0.43.0  node_modules/markdownlint-cli  cesium
rbush               3.0.1   3.0.1   4.0.1  node_modules/rbush             engine@npm:@cesium/engine@12.0.1
rimraf             5.0.10  5.0.10   6.0.1  node_modules/rimraf            cesium
```

- jsdoc - held back https://github.com/CesiumGS/cesium/issues/10455
- lerc - held back https://github.com/CesiumGS/cesium/issues/11034
- markdownlint-cli - updated
- rbush - held back https://github.com/CesiumGS/cesium/issues/12335
- rimraf - held back https://github.com/CesiumGS/cesium/issues/12171







## Issue number and link

No issue, prepping for release





## Testing plan

- Run `rm -rf node_modules`
- Run `npm install`
- Run `npm run markdownlint` just to make sure it still works and there are no errors



# Author checklist
